File: Examen Parcial/20221309_EX1/Pregunta03/TransitSoft/TransitSoftModel/CamaraDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransitSoftModel
{
    public class CamaraDTO
    {
        private int? id;
        private string modelo;
        private string codigoSerie;
        private long? latitud;
        private long? longitud;

        public CamaraDTO()
        {
            this.Id = null;
            this.Modelo = null;
            this.CodigoSerie = null;
            this.Latitud = null;
            this.Longitud = null;
        }

        public CamaraDTO(int? id, string modelo, string codigoSerie, long? latitud, long? longitud)
        {
            this.Id = id;
            this.Modelo = modelo;
            this.CodigoSerie = codigoSerie;
            this.Latitud = latitud;
            this.Longitud = longitud;
        }

        public int? Id { get => id; set => id = value; }
        public string Modelo { get => modelo; set => modelo = value; }
        public string CodigoSerie { get => codigoSerie; set => codigoSerie = value; }
        public long? Latitud { get => latitud; set => latitud = value; }
        public long? Longitud { get => longitud; set => longitud = value; }
    }
}

File: EX2/Conexion RESTful Transitsoft/ServidorRESTFul/TransitSoftModel/src/main/java/pe/edu/pucp/transitsoft/model/CamaraDTO.java
package pe.edu.pucp.transitsoft.model;

public class CamaraDTO {
    private Integer id;
    private String modelo;
    private String codigo_serie;
    private Integer latitud;
    private Integer longitud;

    public CamaraDTO() {
        this.id = null;
        this.modelo = null;
        this.codigo_serie = null;
        this.latitud = null;
        this.longitud = null;
    }

    public CamaraDTO(Integer id, String modelo, String codigo_serie, Integer latitud, Integer longitud) {
        this.id = id;
        this.modelo = modelo;
        this.codigo_serie = codigo_serie;
        this.latitud = latitud;
        this.longitud = longitud;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public String getCodigo_serie() {
        return codigo_serie;
    }

    public void setCodigo_serie(String codigo_serie) {
        this.codigo_serie = codigo_serie;
    }

    public Integer getLatitud() {
        return latitud;
    }

    public void setLatitud(Integer latitud) {
        this.latitud = latitud;
    }

    public Integer getLongitud() {
        return longitud;
    }

    public void setLongitud(Integer longitud) {
        this.longitud = longitud;
    }
}
